### PR TITLE
askama_axum: update axum to 0.7

### PR DESCRIPTION
**Pull request by lictex**
axum-core 0.4

### CODE REVIEW

File: askama_axum/Cargo.toml [hunk 1]
```diff
@@ -15,12 +15,13 @@ readme = "README.md"
...
-http = "0.2"
+axum-core = "0.4"
+http = "1.0"
 
 [dev-dependencies]
-axum = { version = "0.6", default-features = false }
-hyper = { version = "0.14", features = ["full"] }
+axum = { version = "0.7", default-features = false }
+hyper = { version = "1.0", features = ["full"] }
+http-body-util = "0.1"
```
Comment (djc) [inline askama_axum/Cargo.toml:24]: Please add this before hyper so that the dependencies remain in alphabetic order.

**Review by djc — COMMENTED** [review]
Can you add a commit that bumps the version number for askama_axum?

**Comment by lictex** [comment]
bumped to 0.4.0

**Comment by genusistimelord** [comment]
waiting for this to be pushed to crates =3. need it for my axum_csrf library to run tests.

**Comment by djc** [comment]
> waiting for this to be pushed to crates =3. need it for my axum_csrf library to run tests.

Please, my last response is from 5 hours ago. Are you/your org [sponsoring](https://github.com/sponsors/djc) my work yet?

**Comment by genusistimelord** [comment]
I have sponsored in the past but not currently sponsoring. Just waiting is all whenever you get time.

**Review by djc — APPROVED** [review]

**Pull request merged**